_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -79,26 +79,33 @@ release candidates.*
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
 - [Bitcoin Core #14582][] and [#19743][Bitcoin Core #19743] add a new
-  `maxapsfee` ("max avoid partial spends fee") configuration option to allow
-  specifying a maximum fee differential for avoiding partial spends when
-  `avoidpartialspends` is disabled.
-
-  *Enabling `avoidpartialspends` improves privacy by using addresses only once,
-  but it may result in slightly higher fees due to the added constraint of
-  grouping outputs by address. For this reason, `avoidpartialspends` is disabled
-  by default unless `avoid_reuse` is enabled for the wallet. It is for this
-  default case that `maxapsfee` was conceived.*
-
-  The new `maxapsfee` feature allows a choice between three settings. With
-  `-maxapsfee=-1`, partial spend avoidance is completely disabled to optimize
-  for faster fee calculations, which may be useful for very large wallets with
-  many UTXOs. With `-maxapsfee=0` (the default value), fee calculations are made
-  using both coin selection algorithms, and partial spends are avoided if the
-  fees are not higher than regular coin selection. Finally, by setting a
-  `maxapsfee` value greater than `0`, partial spends can be avoided when the
-  absolute fee increase is not greater than the value passed. For example,
-  `-maxapsfee=0.00001000` means the wallet would avoid partial spends if the
-  absolute fee difference is not more than 1000 sats.
+  `maxapsfee` ("max avoid partial spends fee") configuration option to
+  specify the maximum amount of extra fee you're willing to pay to avoid
+  partial spends when the existing `avoidpartialspends` configuration
+  option is disabled.
+
+  Enabling `avoidpartialspends` improves privacy by spending from
+  addresses only once (see [Newsletter #6][news6 avoidpartial]), but it
+  may result in slightly higher fees due to spending all inputs received
+  to the same address when only a subset of those inputs might be needed.
+  For this reason, `avoidpartialspends` is disabled by default unless
+  the `avoid_reuse` flag is enabled for the wallet (see [Newsletter
+  #52][news52 avoid_reuse]).  The new `maxapsfee` feature now gives
```

Comment:
The first paragraph is much better now.

The second paragraph perhaps lost some meaning and transition that was provided by "It is for this default case that `maxapsfee` was conceived", though I'll concede "conceived" may be a bit of an unusual word for the writing here. 

ISTM that "The new `maxapsfee` feature..." is an intro phrase that should be the start of a new paragraph.

I thought that "not more than" and "not greater than" seemed simpler and more concise than "less than or equal to" but deferring to your judgement.

Reply:
Thanks for adding the topic links!

Reply:
@jonatack tried to address your concerns.

The problem I had with "**are not**" was that it always appeared in the same clause as "**are avoided"**.  Having two negatives in the same clause tends to make it hard to parse, at least for a attention deficit like me.  E.g., I found this challenging:

> partial spends are avoided if the fees are not higher than regular coin selection

However, if my edits aren't an obvious improvement, they're not good enough, so I tried again.  Let me know if you think this is better or if I should go back to your original.